dataprofiler/tests/profilers/profiler_options/test_profiler_class_options.py

```diff
@@ -1,143 +1,126 @@
 from dataprofiler.profilers.profiler_options import ProfilerOptions
 from dataprofiler.tests.profilers.profiler_options.test_base_option \
      import TestBaseOption
-
-
```

Comment:
Should put these back

Reply:
In fact it looks like a lot of newlines got deleted, these should all probably remain for readability and PEP8